Guardfile

```diff
@@ -21,3 +21,22 @@ guard :rubocop do
   watch(/.+\.rb$/)
   watch(%r{(?:.+/)?\.rubocop(?:_todo)?\.yml$}) { |m| File.dirname(m[0]) }
 end
+
+guard 'process', name: 'superbot', command: 'superbot' do
+  watch(%r{^lib/(.+)\.rb})
+end
+
+guard 'process', name: 'chromedriver', command: 'chromedriver-helper' do
```

Comment:
This is not needed anymore? I used it for local development. now it should run it with Kommando?